src/mito2/src/worker.rs

```diff
@@ -559,6 +560,16 @@ impl<S: LogStore> RegionWorkerLoop<S> {
                 WorkerRequest::SetReadonlyGracefully { region_id, sender } => {
                     self.set_readonly_gracefully(region_id, sender).await;
                 }
+                WorkerRequest::EditRegion {
+                    region_id,
+                    edit,
+                    tx,
+                } => {
+                    let result = self.edit_region(region_id, edit).await;
+                    if let Err(Err(e)) = tx.send(result) {
+                        warn!("Failed to send edit region error '{e}' to caller!");
```

Comment:
Display will hide some details of the error. You might use debug to print the error or use `warn!(e; "xxx");`
```suggestion
                        warn!("Failed to send edit region error to caller, error: {e:?}");
```